app/static/js/admin/users/users.js

```diff
@@ -11,6 +11,25 @@ $(function() {
         }
     });
 
+    /* =======================================================================
+     * Table Highlight and Click
+     * ==================================================================== */
+    $('td').hover(function() {
+        var t = parseInt($(this).index()) + 1;
+        $('td:nth-child(' + t + ')').addClass('highlighted');
+    },
+    function() {
+        var t = parseInt($(this).index()) + 1;
+        $('td:nth-child(' + t + ')').removeClass('highlighted');
+    });
+
+    var all_td = $('td');
```

Comment:
change this variable name to `$td`.  Explanation: https://github.com/airbnb/javascript#jquery